.github/workflows/go_tests.yml

```diff
@@ -50,6 +50,7 @@ jobs:
       - name: Delete old coverage
         run: "cd sdks && rm -rf .coverage.txt || :"
       - name: Run coverage
+        timeout-minutes: 25
```

Comment:
There is no default step timeout, so I don't think this does what you're expecting. I assume the mentioned timeout was for an individual test or the command itself